common/lib/sandbox-packages/eia.py

```diff
@@ -1,15 +1,113 @@
 """
-Standard resistor codes.
+Standard resistor values.
+
+Commonly used for verifying electronic components in circuit classes are 
+standard values, or conversely, for generating realistic component
+values in parameterized problems. For details, see:
+
 http://en.wikipedia.org/wiki/Electronic_color_code
 """
+
+# pylint: disable=C0103
```

Comment:
Nit: We would like to use the name, rather than the PYlint code, so it's clear what we're disabling. See cleanup work done in #6090. So this would be s/pylint: disable=C0103/pylint: disable=invalid-name/